packages/styles/scss/patterns/sub-patterns/content-item/_content-item.scss

```
@@ -49,6 +49,11 @@
     &__cta {
       margin-top: $carbon--spacing-05;
     }
+
+    .#{$prefix}--video-player {
+      margin-top: 2rem;
+      margin-bottom: 2rem;
```

Looks good @greeshmareddyg, just remember to use tokens for spacing/type values:
https://www.carbondesignsystem.com/guidelines/spacing

```suggestion
      margin-top: $carbon--$spacing-07;
      margin-bottom: $carbon--$spacing-07;
```